File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int n) {
        int m=n;
        int r;
        int reverse=0;
        while(n>0){
            r=n%10;
            n=n/10;
            reverse=reverse*10+r;
        }
        return m==reverse;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if(x < 0) return false;
    let str = x.toString().split("").reverse().join("");
    console.log(str);
    return (x == str);
};